app/app.go

```diff
@@ -318,7 +318,7 @@ func NewSifApp(
 		appCodec, keys[slashingtypes.StoreKey], &stakingKeeper, app.GetSubspace(slashingtypes.ModuleName),
 	)
 	app.CrisisKeeper = crisiskeeper.NewKeeper(
-		app.GetSubspace(crisistypes.ModuleName), invCheckPeriod, app.BankKeeper, authtypes.FeeCollectorName,
+		app.GetSubspace(crisistypes.ModuleName), 1, app.BankKeeper, authtypes.FeeCollectorName,
```

Comment:
Is there a reason to change this to a hardcoded value?

Reply:
Responded in the previous thread